extractContractsInfo.js

```diff
@@ -0,0 +1,38 @@
+const artifactsDir = './lib/build/contracts';
+const dest = './lib/build/abi'
+const fs = require('fs');
+const path = require('path');
+
+const networkId = '1002';
```

Comment:
The previous script was used for kovan and other network, please parameterise it.